src/Main.enso

```diff
@@ -115,84 +203,33 @@ poly_test_2 =
     here.my_function 1 2
     here.mk_object 3 4
 
-## Built-in data types
-data_types =
-    integer = 6
-    decimal = 78.23
-
-    text_sample = "this \n is my text"
-    text_sample_2 = 'this \n is my text'
-    text_block = '''
-        My text
-        can have multiple
-            lines
-        now
-    # `+` on texts is actually fast an preferred
-    concat = text_sample + text_block
-    characters = text.characters
-
-    # Booleans
-    my_true = True
-    my_false = False
-    True && False
-    True || False
-    True.not
-
-    if my_condition then do_something 1 else do_a_different_thing 2
-
-    my_bool = 1 == 2
-    my_bool = 2 > 0.3
-    my_bool = "foo" == "bar"
-
-    # Vectors
-    # create with []
-    my_vec = [1, 2, 3]
-    my_vec_2 = [1, "Foo", False]
-    text.characters # this is also a vector
-    1.up_to 100 . to_vector # [1, 2, ..., 99]
-
-    # Maps
-
-    empty_map = Map.empty
-
-    inserted = empty_map.insert 4 "foo" . insert 5 False
-
-    item = inserted.get 4 # returns "foo"
-    item_2 = inserted.get 10 # returns Nothing (this week) or a Dataflow Error (starting next week)
-
-
-
-## Dataflow Errors (all of this next week)
+## Dataflow Errors 
+# Dataflow errors create a special 
 errors =
-    foo = Error.throw "foo" # Creates a dataflow error
+    # Create a dataflow error:
+    foo = Error.throw "foo" 
 
-    foo.my_method # returns foo
+    # Returns "foo":
```

Comment:
It doesn't really return `"foo"` but "an error with the payload set to `"foo"`. Maybe just change to `Returns foo unmodified, without calling my_method`? 
Same comment relevant throughout this whole section.